### ISSUE TITLE
different Embedding way

### ISSUE THREAD

**Issue by lhbrichard** [graykode/nlp-tutorial#23]
In the code 'Seq2seq-torch.py', i saw u use np.eye,the one-hot representation, to represent embedding, so i change in a normal way ,using nn.Embedding(dict_length,embedding_dim),it can work out. but the loss i got is very high.
i wanna ask the differences between this two ways. here are my code and the result.

![image](https://user-images.githubusercontent.com/33123730/56092521-bba72980-5eef-11e9-8b45-4af1b5cf1c89.png)
![image](https://user-images.githubusercontent.com/33123730/56092549-fc9f3e00-5eef-11e9-9413-772b99e3156f.png)

**Comment by graykode** [comment]
I think `nn.Embedding` is not trainale Parameter because it's not in Model `init()`, also `nn.Embedding` was initialized as randomly...
